src/Workspaces/CoreTest/SolutionTests/SolutionTests.cs

```diff
@@ -2416,7 +2412,7 @@ public void TestGetRecoveredSyntaxRootAsync()
 
             var text = "public class C {}";
 
-            using var workspace = CreateWorkspaceWithRecoverableSyntaxTreesAndWeakCompilations();
+            using var workspace = CreateWorkspace();
```

Comment:
Ditto, shouldn't this fail now?

Reply:
these tests were disabled.  i've mooted them as they don't make any sense now :)